src/capi_frontend/capi.cpp

```diff
@@ -623,34 +643,33 @@ OVMS_Status* OVMS_Inference(OVMS_Server* serverPtr, OVMS_InferenceRequest* reque
         SPDLOG_INFO("Getting modelInstance or pipeline failed. {}", status.string());
         return reinterpret_cast<OVMS_Status*>(new Status(status));
     }
+    // fix execution context and metrics
```

Comment:
missing todo

Reply:
This is included in task tracking already.